ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionAlkoholNarkotikaProvtagningMessage.java

```diff
@@ -56,4 +58,9 @@ public static CertificateDataElement toCertificate(int index, CertificateTextPro
             )
             .build();
     }
+
+    private static Boolean setVisibility(NarkotikaLakemedel narkotikaLakemedel) {
+        final var provtagningBehovs = narkotikaLakemedel != null ? narkotikaLakemedel.getProvtagningBehovs() : null;
```

Comment:
For readability you could consider the comment on QuestionKorrektionsglasensStyrkaMessage.